channels-rke2.yaml

```diff
@@ -5,7 +5,7 @@ releases:
   - version: v1.19.16+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.14+rke2r2
+  - version: v1.20.15-rc1+rke2r1
```

Comment:
Just update this to be non-rc and then LGTM

Reply:
Updated. Thanks!